test/src/kbasesearchengine/test/integration/IndexerWorkerIntegrationTest.java

```diff
@@ -364,61 +369,118 @@ private void checkSearch(
         }
         Assert.assertEquals(1, ids.size());
     }
-    
+
     @Test
-    public void testNarrativeManually() throws Exception {
```

Comment:
Why delete these two tests? Was that intentional?

Reply:
Added them back. PR fatique!